mlflow/langchain/__init__.py

```diff
@@ -911,14 +911,14 @@ def _load_model_from_local_fs(local_model_path, model_config_overrides=None):
         )
 
         try:
+            _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
             model = _load_model_code_path(
                 code_path, {**(model_config or {}), **(model_config_overrides or {})}
```

Comment:
nit: the `code_path` variable name in this line is confusing, could you rename it to `model_code_path` ?